csrc/mmdeploy/apis/csharp/MMDeploy/NativeAPIs/NativeMethods.cs

```diff
@@ -5,6 +5,6 @@
     /// </summary>
     internal static partial class NativeMethods
     {
-        public const string DllExtern = "mmdeploy";
+        public const string DllExtern = "mmdeploy_extern";
```

Comment:
Do we have to make another dll? 
If so, how about `mmdeploy_csharp`? `mmdeploy_extern` sounds ambiguous.